src/test/java/com/algorand/algosdk/integration/Applications.java

```diff
@@ -256,9 +258,15 @@ public void checkAccountData(
         assertThat(found).as("Couldn't find key '%s'", hasKey).isTrue();
     }
 
-    @Then("the contents of the box with name {string} in the current application should be {string}. If there is an error it is {string}.")
-    public void contentsOfBoxShouldBe(String encodedBoxName, String boxContents, String errStr) throws Exception {
-        Response<Box> boxResp = clients.v2Client.GetApplicationBoxByName(this.appId).name(encodedBoxName).execute();
+    @Then("according to {string}, the contents of the box with name {string} in the current application should be {string}. If there is an error it is {string}.")
+    public void contentsOfBoxShouldBe(String from_client, String encodedBoxName, String boxContents, String errStr) throws Exception {
```

Comment:
```suggestion
    public void contentsOfBoxShouldBe(String fromClient, String encodedBoxName, String boxContents, String errStr) throws Exception {
```
camel case for java

Reply:
cAmelStyle

Reply:
It looks like this `from_client` is still in snake case 🐍  is this intentional?

Reply:
fixed.